WordPressAuthenticator/OAuth/ProofKeyForCodeExchange.swift

```diff
@@ -0,0 +1,41 @@
+// https://developers.google.com/identity/protocols/oauth2/native-app#step1-code-verifier
+//
+// FIXME: follow spec!
+//
+// A code_verifier is a high-entropy cryptographic random string using the unreserved
+// characters [A-Z] / [a-z] / [0-9] / "-" / "." / "_" / "~", with a minimum length of 43
+// characters and a maximum length of 128 characters.
```

Comment:
I plan to address the completion of this type in a dedicated PR. The process works for development purposes in the current, plain mode.